### PR TITLE
Updated PR template to say resolves in front of the issue number

### PR DESCRIPTION
**Pull request by karthickbharathi1**
## Issue number

Relevant issue number (e.g. #404): #

## Please check the following

- [ ] Do the tests still pass? (see [Run the Tests](https://github.com/cuttle-cards/cuttle/blob/main/docs/setup-development.md#run-the-tests))
- [ ] Is the code formatted properly? (see [Linting (Formatting)](https://github.com/cuttle-cards/cuttle/blob/main/docs/setup-development.md#linting-formatting))
- For New Features:
  - [ ] Have tests been added to cover any new features or fixes?
  - [ ] Has the documentation been updated accordingly?

### CODE REVIEW

**Review by itsalaidbacklife — COMMENTED** [review]
Thank you for your contribution. Please bump the version field in package.json a patch version and then this is good to go

**Comment by karthickbharathi1** [comment]
So you are telling me to remove the version in the package.json. Am I right?

**Comment by itsalaidbacklife** [comment]
No to update it from 4.3.28 to 4.3.29

**Comment by karthickbharathi1** [comment]
Ok.Thanks

**Review by karthickbharathi1 — COMMENTED** [review]
Hey @itsalaidbacklife I have changed the version from 4.3.28 to 4.3.29.

**Comment by itsalaidbacklife** [comment]
Close, but there are two issues. 
1) The wrong file was updated. We use `package.json`'s version field  not `package-lock.json`
2) there is a merge conflict in package-lock.json. It currently has the git markers for showing two separate versions of the same file where there should only be one

File: package-lock.json [hunk 1]
```diff
@@ -1,12 +1,20 @@
 {
   "name": "cuttle",
-  "version": "4.3.21",
+<<<<<<< Updated upstream
+  "version": "4.3.29",
+=======
+  "version": "4.3.28",
+>>>>>>> Stashed changes
```
Comment (itsalaidbacklife) [inline package-lock.json:7]: This is a merge conflict that needs resolving
```suggestion
  "version": "4.3.29",
```

File: package-lock.json [hunk 2]
```diff
@@ -1,12 +1,20 @@
...
   "requires": true,
   "packages": {
     "": {
       "name": "cuttle",
-      "version": "4.3.21",
+<<<<<<< Updated upstream
+      "version": "4.3.29",
+=======
+      "version": "4.3.28",
+>>>>>>> Stashed changes
```
Comment (itsalaidbacklife) [inline package-lock.json:17]: Same here 
```suggestion
      "version": "4.3.29",
```

**Review by itsalaidbacklife — COMMENTED** [review]
Need to fix the merge conflicts and then update the version field in package.json rather than package-lock

**Comment by seriouslysean** [comment]
@itsalaidbacklife I’ll take a crack at automating the version bump or at least making some sort of npm script to do it more easily so we can document it.

**Comment by itsalaidbacklife** [comment]
Try pulling the latest from the main branch to prevent a conflict in package.json itself

**Review by karthickbharathi1 — COMMENTED** [review]
I have solved the issue @itsalaidbacklife

**Review by itsalaidbacklife — APPROVED** [review]
Nice work. Thanks for your contribution!

**Pull request merged**